spiped/dispatch.c

```diff
@@ -174,7 +178,7 @@ void *
 dispatch_accept(int s, const char * tgt, double rtime, struct sock_addr ** sas,
```

Comment:
The comment here describing the function should be kept in sync with the .h file.